app/helpers/dropdown_helper.rb

```diff
@@ -20,30 +20,38 @@ def audit_status_options
   end
 
   def taxons_options
-    options = @search.options_for("taxons").order(:title)
+    options = ContentItem
```

Comment:
What about inlining this code to:

```ruby
class ContentItem
  def all_taxons
    where(document_type: "taxon").order(:title)
  end
end

taxons = ContentItem.all_taxons
...
```

We could think of a better way to retrieve all taxons in the future, but so far we have that behaviour isolated.